libs/libmicrohttpd/Makefile

```diff
@@ -28,7 +28,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/libmicrohttpd/default
   SECTION:=libs
   CATEGORY:=Libraries
-  TITLE:=GNU libmicrohttpd is a library that runs an HTTP server.
+  TITLE:=GNU libmicrohttpd is a library that provides an HTTP server interface
```

Comment:
Have you verified that it is working with make menuconfig? 🤔 
In my case, the title is not displayed there, because it is too long.

![image](https://github.com/user-attachments/assets/bb66d5d1-220f-48bd-826e-48c95d1fc488)

Reply:
> Compile tested: not tested
Nope :)

Reply:
@neheb Why it is merged then? I don't follow it here.

Reply:
Try with PR #25217